geopandas/tests/test_array.py

```diff
@@ -257,7 +257,9 @@ def test_predicates_vector_scalar(attr, args):
         assert result.dtype == bool
 
         expected = [
-            getattr(tri, attr)(other, *args) if tri is not None else na_value
+            getattr(tri, attr if "geom" not in attr else attr[5:])(other, *args)
```

Comment:
why was this change needed?

Reply:
Because we are passing `attr` `geom_equals` to array, but here we have to pass it to Polygon as `equals`.

Reply:
Ah, yes, of course ;)